### PR TITLE
SNAP-362 Made footer ADA compliant

### PR DESCRIPTION
**Pull request by tparkerosi**
https://osi-cwds.atlassian.net/browse/SNAP-362
Accessibility (ADA) test - Contrast Errors on Snapshot page with very low contrast

## Description
Change font color of footer to increase contrast. 
Add footer class to div container.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
Is a style change.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/assets/stylesheets/accessibility.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import 'colors';
+@import 'constants';
```
Comment (NagaPradeepGajula-osi) [inline app/assets/stylesheets/accessibility.scss]: Was this change intentional or a typo. The styles below use colors. So the import needs to be colors imo.

Comment (wmitchell-cambria) [reply app/assets/stylesheets/accessibility.scss]: It needs to be constants for to include the font-weight variable. Constants in turn includes colors.

Comment (wmitchell-cambria) [reply app/assets/stylesheets/accessibility.scss]: Oh, wait...there's no font-weight variable? Hmm...maybe I'll take that back. It does still include the colors, though.

**Comment by wmitchell-cambria** [comment]
@tparkerosi - Can you take a screenshot of what it looks like now and include it in the story? It helps us to review (although this is very simple), and Implementation will use them to keep tabs on what visual changes are happening as well.

File: app/assets/stylesheets/accessibility.scss (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply app/assets/stylesheets/accessibility.scss]: yeah, it doesn't include font-weight. But thanks anyways I got to know about colors included in constants.

Comment (tparkerosi) [reply app/assets/stylesheets/accessibility.scss]: It turns out the font weight did NOT improve contrast according to wave. I was reading it incorrectly. When I read it, I was hovering over it and it was picking up the visited and reporting it was ok. I change the color to black, but I still used the constant.

**Comment by tparkerosi** [comment]
Updated story with screen shot.
